### PR TITLE
Add EKS update information

### PR DESCRIPTION
**Pull request by jimgus**

### CODE REVIEW

File: docs/xks/operator-guide/kubernetes/eks.md [hunk 1]
```diff
@@ -101,6 +101,133 @@ Run Terraform in the following order:
...
+
+### Update the control plane
+
+Get the name of the cluster to update:
+
+```bash
+aws eks list-clusters --region eu-west-1
+```
+
+Update the control plane version by running the following command with the actual values for \<cluster-name\> and \<version>\:
```
Comment (NissesSenap) [inline docs/xks/operator-guide/kubernetes/eks.md:118]: Is there some command to show the available k8s versions that I can update to?

File: docs/xks/operator-guide/kubernetes/eks.md [hunk 2]
```diff
@@ -101,6 +101,133 @@ Run Terraform in the following order:
...
+aws eks update-cluster-version --region eu-west-1 --name <cluster-name> --kubernetes-version <version>
+```
+
+The above command provides an id that can be use to check the status of the update:
+
+```bash
+aws eks describe-update --region eu-west-1 --name <cluster-name> --update-id <id>
+```
+
+The update is finished when status is `Successful`. Previous updates have approximately taken 45 minutes.
```
Comment (NissesSenap) [inline docs/xks/operator-guide/kubernetes/eks.md:130]: Can you put `45 minutes` to make it super visible.
If I ever have to do this a 3am I don't want to get scared of how long it takes and it should be very clear.

File: docs/xks/operator-guide/kubernetes/eks.md [hunk 3]
```diff
@@ -101,6 +101,133 @@ Run Terraform in the following order:
...
+```bash
+aws eks describe-addon --region eu-west-1  --cluster-name <cluster-name>  --addon-name  <name>
+```
+
+The corresponding Pods can be checked by:
+
+```bash
+kubectl get pods -n kube-system
+```
+
```
Comment (NissesSenap) [inline docs/xks/operator-guide/kubernetes/eks.md:171]: Add section called ### runbook

Add all commands in a row with real examples.

**Review by NissesSenap — COMMENTED** [review]
Small comments

File: docs/xks/operator-guide/kubernetes/eks.md (hunk 3 above)
Comment (NissesSenap) [inline docs/xks/operator-guide/kubernetes/eks.md:171]: Mention that it worth running a general health check.
Maybe curl: https://ingress-healthz.qa.customer.se/

File: docs/xks/operator-guide/kubernetes/eks.md (hunk 1 above)
Comment (jimgus) [reply docs/xks/operator-guide/kubernetes/eks.md:118]: Would have been nice but I didn't find one.

**Review by NissesSenap — APPROVED** [review]
LGTM

**Pull request merged**